x-pack/packages/kbn-elastic-assistant/impl/assistant_context/helpers.ts

```diff
@@ -23,3 +25,34 @@ export const updatePromptContexts = ({
     ...promptContext,
   },
 });
+
+const dataQualityPageExists = (
+  conversations: Record<string, Conversation>,
+  dataQualityConversationId: string
```

Comment:
Would it be possible to remove the `dataQualityConversationId` prop from all the hooks and components, and import `DATA_QUALITY_DASHBOARD_CONVERSATION_ID` wherever we need to use it?